manifests/r/Roblox/RobloxStudio/1.6.3.206/Roblox.RobloxStudio.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.6.3.206
+Installers:
+- InstallerUrl: https://setup.rbxcdn.com/RobloxStudioLauncher.exe
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: D39D8F3E3C98F9059BF359A9C2C56C5565DD71F2F9913D399500A4FCBB9A3CC0
+  InstallerSwitches:
+    Silent: /install /silent
+    SilentWithProgress: /install /silent

```

```suggestion
    Silent: /install silent
    SilentWithProgress: /install silent
```
I'm guessing that the 2nd `/` or `-` might be causing the error.